src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/training
+spring.datasource.username=juan.c.poli
+spring.datasource.password=wolox2021
```

Comment:
Just as a **reminder**, we should not leave credentials in the property files for the production environment. In this case, it is fine. No need to change it